kiwi/system/kernel.py

```diff
@@ -23,7 +23,6 @@
 
 from kiwi.exceptions import KiwiKernelLookupError
 
-
```

Comment:
This line should not be deleted, sorry to be nitpicking, but this causes our `flake` tests to fail.

Reply:
oops, unsure how this crept in ... will fix and update my branch and PR